lib/brains/forms.py

```diff
@@ -16,10 +16,12 @@ def clean_uuid(self):
         try:
             self.buyer = Buyer.objects.get(uuid=data['uuid'])
         except ObjectDoesNotExist:
-            raise forms.ValidationError('Buyer does not exist.')
+            raise forms.ValidationError('Buyer does not exist.',
+                                        code='does_not_exist')
```

Comment:
I think ``buyer_does_not_exist`` would be less ambiguous because there are other entities at play such as sellers and products